### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by TSHEPO-CLOUD**
Add event listeners to list items (dragstart, dragover, drop) and to checkbox (change).
Update items object's values for index and completed keys upon user actions.

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
Hi @TSHEPO-CLOUD, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have linters job for JavaScript. This is a JavaScript project, therefore, its linter configuration (ESlint) must be configured in the `.github/workflows/linters.yml` file. Kindly add the ESlint to the existing job queues in `linters.yml` so as to check your JS scripts.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+- [GitHub](https://github.com/TSHEPO-CLOUD)
+- [Twitter](https://twitter.com/tshepomolefem)
+- [LinkedIn](https://www.linkedin.com/in/tshepo-molefe-8153313b)
+
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
```
Comment (MahmoudBakr23) [inline README.md]: - [ ] Kindly fix the broken issues page to keep your PR professional 😊

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @TSHEPO-CLOUD 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct 
- Great code and file separation 
- Descriptive PR summary and headline 

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄


 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @TSHEPO-CLOUD, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**